### PR TITLE
Add Yeo-Johnson original article as reference to transformer

### PR DESCRIPTION
**Pull request by SangamSwadiK**
Add correct reference as reference 1 and current reference is made as reference 2

### CODE REVIEW

**Comment by SangamSwadiK** [comment]
> Hi @SangamSwadiK
> 
> Thanks a lot for the contribution!
> 
> There are a few small code style things that need fixing. Below what is failing:
> 
> ```
> feature_engine/transformation/yeojohnson.py:72:89: E501 line too long (90 > 88 characters)
> feature_engine/transformation/yeojohnson.py:74:1: W293 blank line contains whitespace
> ```
> 
> You can test locally that the code is OK, by intalling `flake8` and then executing `flake8 feature_engine`
> 
> Would you be able to fix those 2 lines?

Hi @solegalli, I've fixed them.

File: feature_engine/transformation/yeojohnson.py [hunk 1]
```diff
@@ -69,7 +69,11 @@ class YeoJohnsonTransformer(BaseNumericalTransformer):
 
     References
     ----------
-    .. [1] Weisberg S. "Yeo-Johnson Power Transformations".
+    .. [1] Yeo, In-Kwon and Johnson, Richard (2000).
+       A new family of power transformationsto improve normality or symmetry.
```
Comment (solegalli) [inline feature_engine/transformation/yeojohnson.py:73]: almost there... could you please add a space between "transformations" and "to"?

Comment (SangamSwadiK) [reply feature_engine/transformation/yeojohnson.py]: Oops, didn't notice the mistake. I've added proper spacing.
Thanks.

**Comment by solegalli** [comment]
Thank you!!!

**Pull request merged**